src/lightning_lite/connector.py

```diff
@@ -58,6 +59,7 @@
 
 _PLUGIN = Union[Precision, ClusterEnvironment, CheckpointIO]
 _PLUGIN_INPUT = Union[_PLUGIN, str]
+_PRECISION_INPUT = Literal[16, 32, 64, "bf16"]
```

Comment:
I would also include `"16", "32", "64"`

Reply:
Please check my comment below. If I include string alternatives here, now I need to update the comparisons in the connector, which then leads to more typing updates, which then leads to etc. etc.